MonoidalCategories/gap/CoclosedMonoidalCategoriesMethodRecord.gi

```diff
@@ -327,7 +327,30 @@ IsomorphismFromInternalCoHomToObjectWithGivenInternalCoHom := rec(
 
 ) );
 
-CAP_INTERNAL_ENHANCE_NAME_RECORD( COCLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD );
+# collect closed and closed operations
+BindGlobal( "CLOSED_AND_COCLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD", rec( ) );
+
+Perform( RecNames( CLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD ), function ( name )
+    
+    CLOSED_AND_COCLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD.(name) := CLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD.(name);
+    
+end );
+
+Perform( RecNames( COCLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD ), function ( name )
+    
+    CLOSED_AND_COCLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD.(name) := COCLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD.(name);
+    
+end );
+
+CAP_INTERNAL_ENHANCE_NAME_RECORD( CLOSED_AND_COCLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD );
+
+CAP_INTERNAL_GENERATE_DOCUMENTATION_FROM_METHOD_NAME_RECORD(
+    CLOSED_MONOIDAL_CATEGORIES_METHOD_NAME_RECORD,
+    "MonoidalCategories",
+    "ClosedMonoidalCategories.autogen.gd",
```

Comment:
Please merge this and the next line, so that I can replace them simultaneously.

Reply:
Done, but please re-check, I'm not 100% sure I did what you had in mind.